lib/turn.rb

```diff
@@ -0,0 +1,25 @@
+class Turn
+  attr_reader :guess,
+              :card
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    if @guess == @card.answer
+      true
+    else
+      false
+    end
+  end
+
+  def feedback
+    if correct? == true
```

Comment:
Because a call to `correct?` already returns a boolean, you do not need to necessarily compare it to the value `true`. Omitting that part also works.

```ruby
def feedback
  if correct?
    "Correct!"
  else
    "Incorrect."
  end
end
```